### DISCUSSION TITLE
"old-school" implemented liveries disappear with installer

### DISCUSSION THREAD

**Discussion by OddSobriquet** [flybywiresim/aircraft#4105] [Help]
So I have a few liveries that haven't been ported to the "new" way of adding them (stand alone folder in commnity) but rather still use the old method of ammending the layout.json and aircraft.cfg file.
When downloading the a20nx via the installer those liveries disappear, meaning, they aren't offered on the livery selector screen anymore. This seems to be an issue introduced with the new file structure introduced with installer version 1.1.1
Using the github download which still is the pre 1.1.1 file structure (A320NX folder rather than flybywire-... folder) all the liveries remain usable.

Is that a bug or is that a feature?

**Comment by AdamNZ** [comment]
I'd be tempted to re-download those liveries in the new(er) format. You could then use the excellent Addons Linker to enable/disable liveries at will. If there's a favourite livery that you can't find the new version for, let me have a link and I'll see if I can re-jig it for you!
